src/parity_testing/auth/testSuites/fetchAuthSession/Test_that_API_is_called_with_given_payload_and_returns_successful_data.json

```diff
@@ -21,19 +21,20 @@
             "response": {
                 "awsCredentialsResult": {
                     "accessKeyId": "someAccessKey",
-                    "expiration": 2342134,
```

Comment:
Using an expiration for tokens that is way in the future, so that the test cases don't fail.

Reply:
We really dont use expiration value. The method that checks for expiry is mocked and when equating results we have overridden `.equals` to exclude expiration.